projects/plugins/jetpack/extensions/blocks/ai-chat/question-answer.js

```diff
@@ -1,77 +1,40 @@
 /**
  * WordPress dependencies
  */
-import {
-	Button,
-	TextControl,
-	Spinner,
-	KeyboardShortcuts,
-	ExternalLink,
-} from '@wordpress/components';
+import { Button, TextControl, KeyboardShortcuts, ExternalLink } from '@wordpress/components';
 import { RawHTML, useEffect, useState } from '@wordpress/element';
 import { __ } from '@wordpress/i18n';
+import MarkdownIt from 'markdown-it';
 /**
  * Internal dependencies
  */
+import Loading from './loading';
 import useSubmitQuestion from './use-submit-question';
 
-const waitStrings = [
-	__( 'Good question, give me a moment to think about that 🤔', 'jetpack' ),
-	__( 'Let me work out the answer to that, back soon!', 'jetpack' ),
-	__( '🤔 Thinking, thinking, will be back with an answer soon', 'jetpack' ),
-];
+export default function QuestionAnswer() {
+	const [ answer, setAnswer ] = useState( '' );
+	const [ aiResponse, setAiResponse ] = useState( '' );
 
-// This component displays the text word by word if show animation is true
-function ShowLittleByLittle( { html, showAnimation, onAnimationDone } ) {
-	// This is the HTML to be displayed.
-	const [ displayedRawHTML, setDisplayedRawHTML ] = useState( '' );
+	const markdownConverter = new MarkdownIt( { html: true } );
```

Comment:
In theory there should only be markdown in the response from OpenAI, so `html: true` is only needed as we keep parsing the same string, and it turns existing html to html entities if this is not set - but we probably need to have a good think about any security implications, re XSS, etc. around this approach.